File: Python/1161.py
def fat (x):
    valor = 1
    for i in range(1,x+1):
        valor*=i
    return valor;


while True:
    try:
        x = input()
        y = x.split()
        k = [int(i) for i in y]
        print(fat(k[0]) + fat(k[1]))
    except EOFError:
        break
        
 

File: C/1161.c
#include <stdio.h>

long long int fat(int x)
{
    long long int valor = 1;
    for(int i = 1; i <=x ; i++)
    {
        valor*=i;
    }
    return valor;
}

int main()
{
    int m,n;
    while(scanf("%d %d",&m,&n)!=EOF)
    {
        long long int resultado = fat(m) + fat(n);
        printf("%llu\n",resultado);
    }
}
